### ISSUE TITLE
How to use your own dataset to obtain weights that are suitable for the GroundedSAM format

### ISSUE THREAD

**Issue by zy199676** [SysCV/sam-hq#60]
How to use your own dataset to obtain weights that are suitable for the GroundedSAM format

**Comment by lkeab** [comment]
Yes, GroundedSAM has already supported Grounded HQ-SAM. You can see [here](https://github.com/IDEA-Research/Grounded-Segment-Anything/tree/main/EfficientSAM) and their main repo.

**Comment by zy199676** [comment]
> 

How many datasets are suitable for my task
